src/app/components/SkipToContentWrapper.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { HTMLElement } from '@openstax/types/lib.dom';
+import React, { ReactElement } from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import MainContent from './MainContent';
+import SkipToContentWrapper from './SkipToContentWrapper';
+
+// Utility to handle nulls
+function renderToDom<P>(component: ReactElement<P>) {
+    const c = ReactTestUtils.renderIntoDocument(component) as SkipToContentWrapper;
+    if (!c) {
+        throw new Error(`BUG: Component was not rendered`);
+    }
+    const node = ReactDOM.findDOMNode(c);
+    if (!node) {
+        throw new Error(`BUG: Could not find DOM node`);
+    }
+    return {component: c, node};
+}
+
+describe('SkipToContentWrapper', () => {
+
+    it('errors when no main content is provided', () => {
+        const {node} = renderToDom(<SkipToContentWrapper/>);
+        const breaks = () => ReactTestUtils.Simulate.click(node);
+        expect(breaks).toThrowErrorMatchingSnapshot();
```

Comment:
I still get a bunch of red text when a test expects an `Error` to be raised. I noticed another test had suppressed the text... is that really the only way?

Reply:
i didn't realize `.toThrowErrorMatchingSnapshot();` was a thing, but if it shows the error in the output i'd rather not use it... i like the clean output

Reply:
The cause of the console.error messages is from JSDOM trying to be helpful. I suppressed them by adding a `expectError(msg, fn)` which mocks `console.error` when the function runs and then resets `console.error` back.